bdh/api.py

```diff
@@ -0,0 +1,17 @@
+# api.py
+
+from wagtail.api.v2.endpoints import PagesAPIEndpoint
+from wagtail.api.v2.router import WagtailAPIRouter
+from wagtail.images.api.v2.endpoints import ImagesAPIEndpoint
+from wagtail.documents.api.v2.endpoints import DocumentsAPIEndpoint
+
+# Create the router. "wagtailapi" is the URL namespace
+api_router = WagtailAPIRouter('wagtailapi')
+
+# Add the three endpoints using the "register_endpoint" method.
+# The first parameter is the name of the endpoint (eg. pages, images). This
+# is used in the URL of the endpoint
+# The second parameter is the endpoint class that handles the requests
+api_router.register_endpoint('pages', PagesAPIEndpoint)
```

Comment:
Do we need to do any more implementation to fully flesh out the API endpoint, or does it work right out of the box? Also, it might be good to have a chat with @michaelbar98 about how the API endpoints would interface with the database :)